static/js/loginManager.js

```diff
@@ -0,0 +1,10 @@
+const LoginManager = (function() {
```

Comment:
this should be in camelCase starting with lower case

Reply:
Resolved!